.github/CONTRIBUTING.md

```diff
@@ -470,8 +470,6 @@ The `SHOULD_CALL_PARENT(TRUE)` lint should be added to ensure that overrides/chi
 
 * You are expected to help maintain the code that you add, meaning that if there is a problem then you are likely to be approached in order to fix any issues, runtimes, or bugs.
 
```

Comment:
```suggestion
* Do not divide when you can easily convert it to multiplication. (ie `4/2` should be done as `4*0.5`)

```

Reply:
Bad suggestion, this needlessly fgets rid of the empty line 472